### ISSUE TITLE
Class/Custom styles manager 

### ISSUE THREAD

**Issue by jefechang** [elementor/elementor#16271]
## Prerequisites
- [ X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [ X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
Applying the same style to more than one element (es. columns, buttons, anything) it creates a duplication of css code.
At the same time if i have to edit that configuration i'd have to edit it everywhere manually in every page it has been used.

**Describe the solution you'd like**
The possibility to create "custom styles" (aka classes or set of configurations) to be applied to several elements across website.
You select several elements and from a dropdown select your class or configurations set.
In the backend having the possibility to edit these "custom style" centrally.

**Describe alternatives you've considered**
I'm writing my own classes trying to keep in mind where do i put them. At the same time i need to write the code for the responsive part aswell making it even more complicated.
For some project i've switched to oxygen builder that implement these features.

**Additional context**

**Comment by DorShahar** [comment]
Hi @jefechang 
We have few solutions for this:
1. Under Advanced > Advanced there is CSS classes. You can add them manually or use dynamic (like with ACF) 
![image](https://user-images.githubusercontent.com/19154938/132986842-10f3ccd0-b1b0-4505-91da-ccba28cd495a.png)
2. Use global design : under site setting > "Design System" (global colors or global fonts). Each color/font will appear in the relevant control
![image](https://user-images.githubusercontent.com/19154938/132986915-5b11b736-7114-414f-b9c5-7add3a962955.png)

Does it answer your needs?

**Comment by black-eye** [comment]
I think @jefechang might mean something like Oxygen has: https://oxygenbuilder.com/documentation/styling-options/classes/
That would be really useful! I hope Elementor is considering something like this.

**Comment by jefechang** [comment]
Thanks @black-eye, that one linked is the feature i had in mind.
An autocomplete and/or create class option to handle styles.

**Comment by davebeauchemin** [comment]
That's the main feature I am waiting since 2018 with Elementor. I don't understand why it has never been implemented. If you compared with Oxygen or Divi builder, both have Class Base system. 

It would be great if in the advanced tab -> CSS ID/CSS Class we could write a class and if it exist a dropdown show up and we can select the class and then all the styling associated with that CSS class is applied to the widget (padding, margin, background-color, etc.).

The dropdown should be a "mutiple select dropdown" to add more existing classes or add new one.

When you change the font-color on one widget with the same set of class (ex. ".btn .btn-large") then only all widget with the same set of class will be updated (not .btn, only .btn.btn-large) but if I only have .btn on a widget it will overwrite all widget the the .btn class button if they don't have any more specific CSS class and styling.

**Comment by mindingear** [comment]
Yes, if Elementor really wants to be the #1 page builder, we need more design tools to keep a design consistent without having to manually do everything in CSS.
